apps/mobile/src/store/settings/settings.ts

```diff
@@ -0,0 +1,128 @@
+import { useColorScheme } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { whenTheme } from '@/utils/when-theme';
+
+import {
+  AccountDisplayPreference,
+  BitcoinUnit,
+  DefaultNetworkConfigurations,
+  FiatCurrency,
+  WalletDefaultNetworkConfigurationIds,
+} from '@leather.io/models';
+
+import { useAppDispatch } from '../utils';
+import {
+  selectAccountDisplayPreference,
+  selectAnalyticsEnabled,
+  selectAppAuthEnabled,
+  selectBitcoinUnit,
+  selectConversionUnit,
+  selectNetwork,
+  selectTheme,
+  selectWalletSecurityLevel,
+} from './settings.read';
+import {
+  userChangedAccountDisplayPreference,
+  userChangedBitcoinUnit,
+  userChangedConversionUnit,
+  userChangedNetwork,
+  userChangedTheme,
+  userChangedWalletSecurityLevel,
+  userToggledAnalyticsEnabled,
+  userToggledAppAuthEnabled,
+} from './settings.write';
+
+export const defaultNetworks = ['mainnet', 'testnet', 'signet'] as const;
+
+export const defaultThemes = ['light', 'dark', 'system'] as const;
+export type ThemeStore = (typeof defaultThemes)[number];
+export type Theme = Exclude<ThemeStore, 'system'>;
+
+export type WalletSecurityLevel = 'undefined' | 'secure' | 'insecure';
+
+export interface SettingsState {
+  accountDisplayPreference: AccountDisplayPreference;
+  analyticsEnabled: boolean;
+  appAuthEnabled: boolean;
+  bitcoinUnit: BitcoinUnit;
+  conversionUnit: FiatCurrency;
+  createdOn: string;
+  network: DefaultNetworkConfigurations;
+  theme: ThemeStore;
+  walletSecurityLevel: WalletSecurityLevel;
+}
+
+export const initialState: SettingsState = {
+  accountDisplayPreference: 'ns',
+  analyticsEnabled: false,
+  appAuthEnabled: false,
```

Comment:
Can't this be inferred from `walletSecurityLevel`?

Reply:
I need clarification on what this setting is for the from design team. I am unclear what it is, the toast says `Metrics updated`. 🤔 cc @mica000 Can someone clarify for me what `walletSecurityLevel` is for?

Reply:
cc/ @edgarkhanzadian

Reply:
It does appear this is used with biometrics true/false, so I'll remove `appAuthEnabled` bc it seems the same.

Reply:
Just registering that the copy should be App authentication updated.